sorc/ncep_post.fd/MDL2P.f

```diff
@@ -339,11 +338,11 @@ SUBROUTINE MDL2P(iostatusD3D)
                  IF(O3(I,J,1)      < SPVAL) O3SL(I,J)  = O3(I,J,1)
                  IF(CFR(I,J,1)     < SPVAL) CFRSL(I,J) = CFR(I,J,1)
 ! DUST
-                 if (gocart_on) then
-                   DO K = 1, NBIN_DU
-                     IF(DUST(I,J,1,K) < SPVAL) DUSTSL(I,J,K) = DUST(I,J,1,K)
-                   ENDDO
-                 endif
+!                 if (gocart_on) then
```

Comment:
@zhanglikate Since these are legacy code, could they be deleted instead of commented out?

Reply:
@WenMeng-NOAA  Removed. Thanks.